src/Search.php

```diff
@@ -4620,6 +4620,23 @@ public static function addWhere($link, $nott, $itemtype, $ID, $searchtype, $val,
                     break;
             }
         }
+
+        // Is the current criteria on a linked children item ? (e.g. search
+        // option 65 for CommonITILObjects)
+        // These search options will need an additionnal subquery in their WHERE
+        // clause to ensure accurate results
+        // See https://github.com/glpi-project/glpi/pull/13684 for detailed examples
+        $where_on_linked_children = $searchopt[$ID]["use_subquery"] ?? false;
```

Comment:
Variable should probably be renamed now.
```suggestion
        $should_use_subquery = $searchopt[$ID]["use_subquery"] ?? false;
```